### PR TITLE
add new TF for new repo and remove obsolete collaborators from others

### PR DESCRIPTION
**Pull request by MatthewWhiteMoJ**
Add new TF file for new repo and cleanup leavers from other MoJ OFFICIAL Repos

### CODE REVIEW

File: terraform/staff-device-management-intune-scripts-prod.tf [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      review_after = "2022-12-31"                        #  Date after which this grant should be reviewed/revoked, e.g. 2021-11-26
+    },
+    {
+      github_user  = "calkin"
+      permission   = "push"
+      name         = "Josh Calkin"                     #  The name of the person behind github_user
+      email        = "josh.calkin@contentandcloud.com" #  Their email address
+      org          = "Content and Cloud"               #  The organisation/entity they belong to
+      reason       = "PTTP Tech team"                  #  Why is this person being granted access?
+      added_by     = "jgregory@contentandcloud.com"    #  Who made the decision to grant them access? e.g. 'Awesome Team <awesome.team@digital.justice.gov.uk>'
```
Comment (NickWalt01) [inline terraform/staff-device-management-intune-scripts-prod.tf:52]: This line needs a MoJ approver email address

File: terraform/staff-device-management-intune-scripts-prod.tf [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      review_after = "2022-12-31"                        #  Date after which this grant should be reviewed/revoked, e.g. 2021-11-26
+    },
+    {
+      github_user  = "LiliNedeleva"
+      permission   = "admin"
+      name         = "Lili Nedeleva"                     #  The name of the person behind github_user
+      email        = "lili.nedeleva@contentandcloud.com" #  Their email address
+      org          = "Content and Cloud"                 #  The organisation/entity they belong to
+      reason       = "PTTP Tech team"                    #  Why is this person being granted access?
+      added_by     = "jgregory@contentandcloud.com"      #  Who made the decision to grant them access? e.g. 'Awesome Team <awesome.team@digital.justice.gov.uk>'
```
Comment (NickWalt01) [inline terraform/staff-device-management-intune-scripts-prod.tf:42]: This line needs a MoJ approver email address

File: terraform/staff-device-management-intune-scripts-prod.tf (hunk 1 above)
Comment (MatthewWhiteMoJ) [reply terraform/staff-device-management-intune-scripts-prod.tf:52]: one sec let me review

File: terraform/staff-device-management-intune-scripts-prod.tf (hunk 2 above)
Comment (MatthewWhiteMoJ) [reply terraform/staff-device-management-intune-scripts-prod.tf]: should now be resolved

File: terraform/staff-device-management-intune-scripts-prod.tf (hunk 1 above)
Comment (MatthewWhiteMoJ) [reply terraform/staff-device-management-intune-scripts-prod.tf]: should now be resolved

**Review by NickWalt01 — APPROVED** [review]

**Pull request merged**